galasa-parent/galasa-boot/src/main/java/dev/galasa/boot/Launcher.java

```diff
@@ -435,7 +437,7 @@ private void checkForLocalMaven(CommandLine commandLine) {
         } else {
             try {
                 this.localMavenRepo = new File(
-                        System.getProperty(USER_HOME) + File.separator + ".m2" + File.separator + "repository").toURI()
+                        new SystemEnvironment().getProperty(USER_HOME) + File.separator + ".m2" + File.separator + "repository").toURI()
```

Comment:
This looks a bit suspect ? Shouldn't an environment it be passed in ?
How can it be mocked-out otherwise ?

Reply:
Added the Environment interface to the class as a public member and pass that instead.  good spot